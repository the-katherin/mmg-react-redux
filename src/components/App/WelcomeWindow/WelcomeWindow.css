.welcome-window {
  display: grid;
  justify-content: center;
  justify-items: center;
  grid-row-gap: 20px;
  padding: 30px;
  border-radius: 10px;
  border: 1px solid white;
  background: rgba(0, 120, 201, 0.5);
  
}

.game-rules {
    display: grid;
    grid-row-gap: 10px;
    justify-content: center;
    justify-items: center;

}








