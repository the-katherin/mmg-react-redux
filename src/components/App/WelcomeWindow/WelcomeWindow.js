import React from 'react';
import { Link } from "react-router-dom";

const WelcomeWindow = () => (
    <div className="welcome-window">
        <h1>Welcome to Match-Match Game!</h1>

        <div className="game-rules">

            <h2>How to play:</h2>
            <p>Memory is a counter game where the object is to find pairs.</p>

            <h2>To Play:</h2>
            <ol>
                <li>Select two cards to try to match the pictures.</li>
                <li>If you match the pictures you can go again.</li>
                <li>If they don't match the computer turns them back.</li>
                <li>The player who finds Ð°ll pairs wins!</li>
                <li>Have Fun!</li>
            </ol>

        </div>

        <Link to='/form'>
            <button>
                Got it!
             </button>
        </Link>
    </div>
);

export default WelcomeWindow;