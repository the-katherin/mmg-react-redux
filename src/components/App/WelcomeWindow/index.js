import './WelcomeWindow.css';

export { default } from './WelcomeWindow';