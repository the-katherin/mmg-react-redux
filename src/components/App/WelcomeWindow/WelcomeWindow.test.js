import React from "react";
import WelcomeWindow from "./WelcomeWindow";

describe("WelcomeWindow", () => {
    test("match snapshot", () => {
        const wrapper = shallow(<WelcomeWindow />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

});