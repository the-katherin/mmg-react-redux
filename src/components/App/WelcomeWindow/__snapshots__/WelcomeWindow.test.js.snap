// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeWindow match snapshot 1`] = `
<div
  className="welcome-window"
>
  <h1>
    Welcome to Match-Match Game!
  </h1>
  <div
    className="game-rules"
  >
    <h2>
      How to play:
    </h2>
    <p>
      Memory is a counter game where the object is to find pairs.
    </p>
    <h2>
      To Play:
    </h2>
    <ol>
      <li>
        Select two cards to try to match the pictures.
      </li>
      <li>
        If you match the pictures you can go again.
      </li>
      <li>
        If they don't match the computer turns them back.
      </li>
      <li>
        The player who finds Ð°ll pairs wins!
      </li>
      <li>
        Have Fun!
      </li>
    </ol>
  </div>
  <Link
    replace={false}
    to="/form"
  >
    <button>
      Got it!
    </button>
  </Link>
</div>
`;
